src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -333,19 +333,19 @@ public Visitable visit(ClassOrInterfaceDeclaration node, Void arg) {
     typeVariables.removeFirst();
 
     NodeList<ClassOrInterfaceType> interfaceList = node.getImplementedTypes();
-    for (ClassOrInterfaceType interfaceType : interfaceList) {
+    // Not sure why getExtendedTypes return a list, since a class can only extends at most one class
+    // in Java.
```

Comment:
Does it return the entire list of superclasses? If so, that would be useful to us!

(That is, for `class A extends B`, where `class B extends C`, does calling `getExtendedTypes()` on `A` lead to `[B, C]`?)

Reply:
I have just tested that method with a chain of inheritance. Sadly, it doesn't return the entire list. It only returns the immediate superclass.

Reply:
Too bad. You can ignore my comment, then.